### ISSUE TITLE
Invalid source for aria-labelledby of Select's listbox in Chrome

### ISSUE THREAD

**Issue by mxp-qk** [adobe/react-spectrum#9437]
### Provide a general summary of the issue here

The specified id for the aria-labelledby attribute of a listbox in the Select component is marked as an invalid source in Chrome.

This issue seems to concern Chrome only

### 🤔 Expected Behavior?

It should be marked as valid and the listbox named after the select label

### 😯 Current Behavior

The aria-labelledby properties is invalid and the listbox doesn't have a name

### 🔦 Context

I had some test selectors that use the listbox role and name that broke with the last version.
```
.page()
.getByRole("listbox", { name: "Select label" })
.getByRole("option", { name: "Option 1" })
```

### 🖥️ Steps to Reproduce

Inspecting the listbox in the [documentation page](https://react-aria.adobe.com/Select) for the Select component shows an invalid computed properties.

<img width="348" height="230" alt="Image" src="https://github.com/user-attachments/assets/b04cc5d6-076e-44f3-8f65-860bfa302362" />

### Version

react-aria-components@1.14.0

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

MacOS

**Comment by snowystinger** [comment]
Odd, it is found by `document.querySelector`, and it's valid for other elements. All I can think is that the inert ancestor is causing it to mark it like that. But it still doesn't for the button.

Are there any chrome bugs reported about this?

**Comment by mxp-qk** [comment]
I did not find specific issue for this but similar one [here](https://issues.chromium.org/issues/41487406) where it concerns MacOs only or [here](https://issues.chromium.org/issues/396169350) where the issue was on windows instead.

> All I can think is that the inert ancestor is causing it to mark it like that. But it still doesn't for the button.

Yes it is caused by the button or label being inert. This [codepen](https://codepen.io/mxp-qk/pen/PwzNBOj) has the same issue on Chrome on macOS where the aria-labelledby references the inert button ID and is marked as an invalid source. It works fine on Firefox and Safari on macOS.

**Comment by snowystinger** [comment]
Thanks for that codepen, technically nothing is explicitly marked inert there. But this makes me think it's a Chrome bug, would you mind opening an issue with them?

**Comment by mxp-qk** [comment]
The issue is [here](https://issues.chromium.org/issues/324211424).
I don't think this issue needs to stay open.

As an alternative for now, here the selector I use instead : 
```typescript
.page()
// .getByRole("listbox", { name: "Select label" })
.getByRole("dialog") // Ensure the listbox selector targets the Select dialog only
.getByRole("listbox")
```

**Comment by snowystinger** [comment]
Thanks for checking, will close
